### PR TITLE
Added Prettier to the features in devcontainer

### PR DESCRIPTION
**Pull request by NicoleHeather**
What is Prettier?

An opinionated code formatter
Supports many languages
Integrates with most editors
Has few options
Why?

Your code is formatted on save
No need to discuss style in code review
Saves you time and energy
And more

### CODE REVIEW

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

File: docker-compose.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ services:
     volumes:
       - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
       - ${HOME}/.ssh:/home/ubuntu/.ssh
-      - ${PWD}:/app
+      - $./:/app
```
Comment (growlf) [inline docker-compose.yml:18]: Typically we use the shell variable for cross system compatibility.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -34,7 +34,10 @@
 			"installTools": true,
 			"version": "3.11"
 		},
-		"ghcr.io/warrenbuckley/codespace-features/sqlite:1": {}
+		"ghcr.io/warrenbuckley/codespace-features/sqlite:1": {},
+		"ghcr.io/devcontainers-community/features/prettier": {
```
Comment (growlf) [inline .devcontainer/devcontainer.json:38]: Nice addition!

**Review by growlf — APPROVED** [review]

**Comment by growlf** [comment]
This PR is generally GOOD! If it was against the correct development branch, we would accept it.  Please recommit it and target the `ensf400_w24` branch and we will accept it.

**Pull request closed without merging**